gui/src/components/preboarding/license-agreement.vue

```diff
@@ -81,7 +81,7 @@
                 class="cortx-btn-tertiary"
                 @click="cancelAgreement"
               >
-                {{ $t("common.cancel") }}
+                {{ $t("admin.cancel") }}
```

Comment:
We can leave out common related translations, please revert all common related changes.

Reply:
done